File: app/src/main/java/com/example/nasaenterpriseapi/Model/Links.java
package com.example.nasaenterpriseapi.Model;

import com.google.gson.annotations.SerializedName;

public class Links {

	@SerializedName("rel")
	private String rel;

	@SerializedName("href")
	private String href;

	@SerializedName("render")
	private String render;

	public String getRel(){
		return rel;
	}

	public String getHref(){
		return href;
	}

	public String getRender(){
		return render;
	}
}

File: app/src/main/java/com/example/nasaenterpriseapi/model/NasaQueryResponse/Links.kt
package com.example.nasaenterpriseapi.model.NasaQueryResponse

import com.google.gson.annotations.SerializedName

data class Links (

	@SerializedName("render") val render : String,
	@SerializedName("href") val href : String,
	@SerializedName("rel") val rel : String
)